### ISSUE TITLE
[FR]: Increase the frequency range of bands, esp low pass and high shelf? 

### ISSUE THREAD

**Issue by SionnachDearg** [ZL-Audio/ZLEqualizer#140]
### Thanks for taking the time to make a feature request!

- [X] I have read the plugin manual and searched the existing issues.

### Why?

```markdown
Hello, thank you for this great eq. 

I use a low pass (high cut) filter at 6db/oct or 12 fairly often, to slightly tip high frequencies downwards at the air frequencies. I would love to be able to do this for this eq, but it requires me to set a low pass as high as 40 khz, usually more like 30-35.
```

### What do you want?

```markdown
That way it JUST starts to tip down in the audible bandwidth. 

I use shelves this way too sometimes, setting them above 20khz. 

Is it possible to implement something like this? I guess it might require oversampling the eq a bit? 

I mostly mix in 88.2khz to have the antialiasing filter higher, otherwise the filter cramps when I'm doing this. 

Thank you!
```

### Code of Conduct

- [X] I agree to follow the Code of Conduct.

**Comment by zsliu98** [comment]
If you want a low pass at 30kHz under 44.1/48kHz sample rate, the plugin needs:

- either oversampling: for example, TDR Nova, which introduces latency and increases CPU load. I want to avoid it if possible.
- or a robust de-cramping filter algorithm: for example, Pro-Q (but even Pro-Q fails to do it accurately when the slope of the filter is high). Although this plugin also has built-in de-cramping, I am not able to make it work when the cut-off freq is above or very close Nyquist.

 If you want to slightly tip down the air frequency, you could use a 24dB/oct high-shelf at 20kHz.

**Comment by SionnachDearg** [comment]
I should’ve checked, is it possible to set it at 30khz when using higher
sample rate sessions? because that’s what im wondering about- 88.2 and above

**Comment by zsliu98** [comment]
Yes, it is possible. Under 88.2kHz we can set it at 30kHz without over-sampling as 30kHz is under Nyquist.

However, the workload is too huge, as I have to rewrite a lot of UI code. I won't include it in my current plan.

**Comment by SionnachDearg** [comment]
thank you for considering it! It would be very helpful, very grateful for
your work.

**Comment by zsliu98** [comment]
I have changed the max frequency to:

| Samplerate    | Max Freq |
| -------- | ------- |
| 44100 | 20000 |
| 48000 | 21800 |
| 88200 | 40000 |
| 96000 | 43600 |
| ...   | ...   |

It has not been released yet, but it will be included in the next version (ZL Equalizer 2).
